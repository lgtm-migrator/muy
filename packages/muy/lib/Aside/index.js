import React from "react"
import Box from "@material-ui/core/Box"

const Aside = ({ display, ...props }) => (
  <Box component={"aside"} display={display} {...props} />
)

export default Aside

Aside.defaultProps = {
  display: "block",
}
