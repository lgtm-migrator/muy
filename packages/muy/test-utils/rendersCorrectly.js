import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
