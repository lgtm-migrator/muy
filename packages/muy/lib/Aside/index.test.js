import React from "react"
import Aside from "."
import rendersCorrectly from "../../test-utils/rendersCorrectly"

describe("<Aside /> component", () => {
  it("renders correctly", () => {
    rendersCorrectly(<Aside />)
  })
})
