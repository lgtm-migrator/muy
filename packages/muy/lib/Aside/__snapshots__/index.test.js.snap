// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Aside /> component renders correctly 1`] = `
<aside
  className="MuiBox-root MuiBox-root-2"
/>
`;
